src/orquestra/sdk/_base/_logs/_regrouping.py

```diff
@@ -14,8 +14,10 @@
 # 903b5bfa866935b17bbd3d3a64d22c7c6b746c2d522ad5fca9cf1f6a-01000000-249.err"
 WORKER_GLOB = "session_*/logs/worker*.???"
 
-# Example: "/tmp/ray/session_latest/logs/runtime_env_setup-01000000.log"
-ENV_SETUP_GLOB = "session_*/logs/runtime_env_setup-*.log"
+# Examples:
+# "/tmp/ray/session_latest/logs/runtime_env_setup-01000000.log"
+# "/tmp/ray/session_latest/logs/runtime_env_actor.log"
+ENV_SETUP_GLOB = "session_*/logs/runtime_env_*.log"
```

Comment:
**issue**: per @jamesclark-Zapata's [point](https://zapatista.slack.com/archives/C02DN7BA04E/p1692935689703889?thread_ts=1692880954.476179&cid=C02DN7BA04E), we shouldn't include this file in the logs. Let's discuss our options before continuing

Reply:
I see 5 potential routes (listed in descending order of my preference):
1. We know that the `ray.exceptions.RuntimeEnvSetupError`, while not raised during `get()`, _is_ printed to stdout at least on local Ray- it's just not captured in any of the logs. If we can capture it somehow we can use that to determine the message rather than parsing logs.
2. There may be other telltales in the logs that I missed, and that don't entail a security risk. 
3. Infer env setup problems by the details of other logs, essentially try to identify the negative space.
4. We could investigate handling the logs in such a way that  this detail of `runtime_env_agent.log` is available internally but the rest remains shuttered. Some function at a much lower level that strips any line that isn't the one(s) we're looking for perhaps?
5. Migrate to Ray primitives.

5 is massively beyond the scope we originally intended for this; 4 is dangerous if we aren't perfect; 3 would require quite a lot of assumptions but should work, and the consequences for an incorrect inference is that the user gets a less precise message; 2 would be easy but is unlikely; 1 _should_ be possible and would give us certainty when this sort of problem has happened.

WDYT?

Reply:
Consideration on 1: I've had impression that the `RuntimeEnvSetupError` error message is only printed to stdout/err only if the driver process (the process that submitted a workflow to Ray) lives long enough to see it. A Python script with a `.wait_until_finished()` poll: yes, a submit-and-exit script or `orq wf submit`: no. Am I correct?

Reply:
That matches my understanding, yes.